README.md

```diff
@@ -104,6 +106,8 @@ in preferred order.
 Enable or disable `Last-Modified` header, defaults to true. Uses the file
 system's last modified value.
 
+This will default to `false` if the [`transform` option is passed](#transform).
```

Comment:
This wording is misleading; if the `transform` option is passed, but as `transform: undefined`, I assume this will not default to `false`, but the wording says it will as long as the option is _passed_ (without regard to what value is passed).